src/openrct2/localisation/Formatting.cpp

```diff
@@ -289,11 +291,15 @@ namespace OpenRCT2
     {
         if (IsRealNameStringId(id))
         {
-            auto realNameIndex = id - kRealNameStart;
-            ss << real_names[realNameIndex % std::size(real_names)];
-            ss << ' ';
-            ss << real_name_initials[(realNameIndex >> 10) % std::size(real_name_initials)];
-            ss << '.';
+            auto& objManager = GetContext()->GetObjectManager();
+            auto* peepNamesObj = static_cast<PeepNamesObject*>(objManager.GetLoadedObject(ObjectType::PeepNames, 0));
```

Comment:
Should we add a `Guard` here to make sure this does not silently break?

Reply:
There is no legitimate way to get to a state where no peep object is loaded. If you somehow get the game in that state, I believe the object manager will complain/throw already.